src/main/java/org/opentripplanner/api/common/RoutingResource.java

```diff
@@ -88,14 +88,14 @@
 
     ///////////////////////////////////////////////////////////////////////////////////////////////
     /** NIH routing paramters */
-    @DefaultValue("true") @QueryParam("allowCrossSlope") protected Boolean allowCrossSlope;
-    @DefaultValue("false") @QueryParam("restingPlaces") protected Boolean restingPlaces;
-    @DefaultValue("false") @QueryParam("usingWalkerCane") protected Boolean usingWalkerCane;
+    @DefaultValue("true") @QueryParam("allowCrossSlope") protected List<Boolean> allowCrossSlope;
```

Comment:
I'm not entirely sure, but the other parameters are lists.  I'm guessing it may have to do with having intermediate destinations.